01-cloudformation/README.md

```diff
@@ -160,19 +160,34 @@ name.
 
 _Why do we prefer the YAML format for CFN templates?_
 
+#### The intrinsic functions
```

Comment:
YAML inherently provides some features, such as commenting, that aren't available in JSON. Also with JSON, you must account for curly brackets; why give yourself more formatting work when you don't need to?

The lack of extra delimiters also makes YAML a bit more lightweight than JSON. Indenting has proved to be a very minimal and simple way of seperating concerns, and works really well. Another proof of this point apart from YAML is the Python programming language.

Lastly, YAML is a superset of JSON. So a YAML compiler will actually be able to parse JSON, however not the other way around.